ultravox/tools/tts_tool.py

```diff
@@ -0,0 +1,73 @@
+import argparse
+import os
+
+import datasets
+
+from ultravox.tools import tts
+
+# This script is used to generate audio samples from text using a TTS model.
+# Ex: python tts_tool.py -d google/boolq -c question -a audio -u "fixie-ai/boolq-audio
```

Comment:
Does that mean we can incrementally add other fields too if need be? For example convert passage as well?

Reply:
Yep, this is just a v1. We could perhaps allow specifying multiple columns to be converted in a follow up PR.

Reply:
Yeah I meant this already enables that which is neat!